src/pages/index.vue

```diff
@@ -31,20 +31,70 @@
     color="danger"
     icon-name="delete"
   ></IconButton>
+
+  <div class="course-grid">
+    <CourseTile
+      @click="tileStat = tileStat == 'active' ? 'default' : 'active'"
+      :state="tileStat"
+      course-name="学校を考える"
+      course-id="1A101"
+    ></CourseTile>
+
+    <CourseTile
+      state="active"
+      course-name="学校を考える"
+      course-id="1A101"
+    ></CourseTile>
+
+    <CourseTile state="none"></CourseTile>
+
+    <CourseTile
+      state="default"
+      course-name="学校を考える"
+      course-id="1A101"
+      caution="他1件"
+    ></CourseTile>
+  </div>
+
+  <div class="course-details">
+    <CourseDetail
+      iconName="schedule"
+      item="開講時限"
+      value="春AB 木2"
+    ></CourseDetail>
+
+    <CourseDetail
+      iconName="person"
+      item="担当教員"
+      value="山本早里"
+    ></CourseDetail>
+
+    <CourseDetail iconName="room" item="授業場所" value="6A508"></CourseDetail>
+
+    <CourseDetail
+      iconName="category"
+      item="授業形式"
+      value="対面"
+    ></CourseDetail>
+  </div>
 </template>
 
 <script lang="ts">
 import { defineComponent, ref } from "vue";
 import Button from "../components/Button.vue";
 import IconButton from "../components/IconButton.vue";
 import ToggleIconButton from "../components/ToggleIconButton.vue";
+import CourseTile from "../components/CourseTile.vue";
```

Comment:
CourseTile.vue側で`export type State = ...`して

```suggestion
import CourseTile, {
  State as CourseTileState,
} from "../components/CourseTile.vue";
```

106行目を

```
const tileStat = ref<CourseTileState>("default");
```

が型安全